HISMalawi/BHT-EMR-API
# [ART] Added pDTG regimens to cohort report.

Update adds indicators for regimen 14PP and 15PP to the MoH cohort report.

## Steps to test

To make things easy, you may want to do this on a blank database but it's not necessary, however you will have to jump step 5 and record the numbers so that you compare afterwards.

1. Using the ART app create a patient on any day of a given quarter
2. Give the patient a weight of not more than 20 kilograms
3. Prescribe either 14PP or 15PP to them
4. Complete patient visit
5. Using whatever tool make a GET /api/v1/programs/1/reports/cohort?name=custom&start_date=YYYY-MM-DD&end_date=YYYY-MM-DD&regenerate=true
    - Make sure the period specified in the report above covers the date you registered the patient
6. From the response of the above request, look for a report_value with name fourteen_pp or fifteen_pp (depending on what was dispensed to the patient) and check if the patient has been included.

--
Closes #63.
